.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+**Fixes**
+
+<!-- If PR only partly solves the issue, replace 'Fixes' with 'Fixes part of' -->
+
+Fixes #[issue number] by @[issue author]
+
+**Description**
+
+<!-- A clear and concise description of what the pull request does. -->
+
+**Checklist:**
+
+<!-- Replace  the [ ] with [x] to check the boxes -->
+
+-   [ ] I have read the contribution guidelines in the repository.
+-   [ ] My pull request has a descriptive title (not a vague title like "Update package.json").
+-   [ ] My code follows the established code style of the repository.
+-   [ ] I have tried running the project locally and verified that there are no visible errors.
```

Comment:
We can add something like my branch is in sync with main branch too rest LGTM

Reply:
How about "My branch is in sync with the branch this PR targets"?

Reply:
All will target main right now 🤔 we can have something simple like "My branch is up-to-date with upstream branch"

Reply:
Yeah right!